content/english/debugging/05-fixing-the-problem.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Step 2 - Fixing the problem"
+difficulty: "Intermediate"
+weight: 1

```

Comment:
Please adjust the weights so that they are numbered in order of how the pages should be ordered